tasks/eva_2357/populate_ids.py

```diff
@@ -0,0 +1,196 @@
+import hashlib
+import argparse
+import pymongo
+import traceback
+import logging
+from ebi_eva_common_pyutils.config_utils import get_mongo_uri_for_eva_profile
+
+
+def generate_update_statement(hash_to_variant_ids, hash_to_accession_info):
+    variant_to_ids = {}
+    update_statements = []
+    for sve_hash, accessions in hash_to_accession_info.items():
+        ss_accession = f"ss{accessions.get('ss')}"
+        rs_accession = f"rs{accessions.get('rs')}" if accessions.get('rs') else accessions.get('rs')
+        variant_id = hash_to_variant_ids.get(sve_hash)
+
+        # Group all the ids (ss and rs) by variant id
+        if variant_to_ids.get(variant_id):
+            variant_to_ids[variant_id].add(ss_accession)
+            if rs_accession:
+                variant_to_ids[variant_id].add(rs_accession)
+        else:
+            ids = set()
+            ids.add(ss_accession)
+            if rs_accession:
+                ids.add(rs_accession)
+            variant_to_ids[variant_id] = ids
```

Comment:
Could this be shortened to?
```suggestion
            variant_to_ids[variant_id] = [ss_accession] + [rs_accession] if rs_accession else []
```